### ISSUE TITLE
Runtime error trying to backward through the graph a second time, but the buffers have already been freed.

### ISSUE THREAD

**Issue by agsourav** [agsourav/tweet_sentiment_analysis#1]
**retain_graph** has been set to **True** in loss.backward() . tried with hidden.detach_() as well in between iterations after init_hidden().
